tests/integration/features/command/command-list.feature

```diff
@@ -0,0 +1,21 @@
+Feature: command/command-list
+
+  Scenario: Get the output options
+    Given invoking occ with "talk:command:list --help"
+    Then the command was successful
+    And the command output contains the text "Output format"
+
+  Scenario: List all available commands as plain text (default)
+    Given invoking occ with "talk:command:list"
+    Then the command was successful
+    And the command output contains the text "Response values: 0 - No one"
```

Comment:
Didi you prefer to remove this scenario and check only if have the json option?

Reply:
```suggestion
    And the command output contains the text "| help  "
```

Reply:
I verified the header because this only exist at plain text output. We can't have this output with json format.

Reply:
> I verified the header because this only exist at plain text output. We can't have this output with json format.

Just do both?

```suggestion
    And the command output contains the text "Response values: 0 - No one"
    And the command output contains the text "| help  "
```